README.md

```diff
@@ -232,7 +232,7 @@ Leave this as `default`.
 The title of the page. Quoting is only necessary if the title contains YAML
 keywords (like "on").
 
-    published: true|false
+    date: 2025-05-22T00:00:00+00:00|false
```

Comment:
So, date is the date timestamp after which a build would publish the page. Is that date used by anything else?